### ISSUE TITLE
Bug - HDG "pull" behaviour

### ISSUE THREAD

**Issue by andras** [legoboyvdlp/A320-family#251]
**Describe the bug**
I believe the AP behaves wrong when turning the HDG selector while it's pulled.

**To Reproduce**
Steps to reproduce the behaviour:
1. Take off, fly HDG 360.
2. Pull HDG selector.
3. Turn HDG clockwise to 270.
4. Aircraft will first turn right, then change direction when the HDG bug is rotated past the reciprocal of current heading.

**Expected behaviour**
I believe the AP, when HDG is pulled, should cause the aircraft to turn in the same direction the HDG knob was turned first after pulling it.  This is to enable flying turns larger than 180 degrees (as in ATC request "turn right heading 270" while you were flying heading 360 before).  This is in contrast to setting the new heading while HDG is pushed, then pulling HDG, which makes the aircraft always turn to the new heading the "short way".

**System (please complete the following information):**
 - OS: Linux
 - FlightGear version: 2020.3.6
 - A320 Version: current HEAD from master branch

**Comment by merspieler** [comment]
Do you have any documentation for this?

**Comment by Octal450** [comment]
Known thing, it's on my autopilot/FBW roadmap.

Kind Regards,
Josh

**Comment by andras** [comment]
Thank you Josh, I just started searching through all the Airbus PDFs I collected a while ago (>50k pages or so in total) for the docs.  Now I know my memory was right and I no longer need to bother...

**Comment by merspieler** [comment]
mind sharing those docs? maybe you've got something we haven't.
If so, write me on the forum.

**Comment by Octal450** [comment]
Yes please share the docs if you have. I only know it does this but not specifics.

Kind Regards,
Josh

PS: Also, I don't have an ETA for this right now, I'm trying to sort my medical issues.
